controllers/edit_profile.php

```diff
@@ -1,3 +1,50 @@
 <?php
+function loopGetinfo($sql)
+{
+    foreach ($sql as $statementresults => $statementresult) {
+        foreach ($statementresult as $results) {
+            return $results;
+        }
+    }
+}
+function updateUserInfo()
+{
+    return [
+        "firstName" => loopGetinfo(customStatement("SELECT first_name FROM users WHERE " . $_SESSION['users_id'] . " = users.users_id")),
+        "lastName" => loopGetinfo(customStatement("SELECT last_name FROM users WHERE " . $_SESSION['users_id'] . " = users.users_id")),
+        "email" => loopGetinfo(customStatement("SELECT email FROM users WHERE " . $_SESSION['users_id'] . " = users.users_id")),
+    ];
+}
+
+$_SESSION['users_id'] = 1;
+$target_dir = $_SERVER['DOCUMENT_ROOT'] . "/views/public/images/";
+$user_id = $_SESSION['users_id'];
+if (isset($_POST['edituser'])) {
+    customStatement("UPDATE users SET first_name = '" . $_POST['firstName'] . "' WHERE " . $user_id . " = users.users_id");
```

Comment:
maak hier even van: 

`UPDATE users SET first_name = '" . $_POST['firstName'] . "', last_name = '" . $_POST['lastName'] . "', email= '" . $_POST['email'] . "' WHERE users.users_id = '.$user_id.'"`;

Dan hoeft mysql maar 1 update query uit te voeren. ( Geen idee of de syntax correct is trouwens :P maar mysql kan meerdere columns updaten tegelijkertijd. ). en again sql injection

Reply:
klopt, helemaal niet aan gedacht! goed gezien